indent/verilog_systemverilog.vim

```diff
@@ -169,12 +173,9 @@ function GetVerilog_SystemVerilogIndent()
   " or if it starts with a `define call, which might not require the ; end
   elseif (
     \ last_line =~ ';' . vlog_comment . '*$' ||
-    \ last_line =~ '^\s*`\k\+'
-    \ ) &&
-    \ last_line2 =~ '\<\(`\@<!if\|`\@<!else\|for\|while\|always\|initial\|do\|foreach\|final\)\>\(\s*(.*)\)\?\s*' . vlog_comment . '*$' &&
-    \ last_line2 !~ '\(//\|/\*\).*\<\(`\@<!if\|`\@<!else\|for\|while\|always\|initial\|do\|foreach\|final\)\>' &&
```

Comment:
I thought this line was unnecessary as ```last_line2``` excludes any lines that are commented out. The only situations I can think that this breaks is when there is ````end //else if()```, ```end // else``` and ```end // while();```. However the line was already broken if you have ```/* comment */ always```. It also doesn't support block labels but I'll sort that in another PR. For now I'll add the lines back in and I'll add some code to the test that actually exercise these.